src/test/java/org/needle4j/configuration/PostConstructExecuteStrategyTest.java

```diff
@@ -0,0 +1,23 @@
+package org.needle4j.configuration;
+
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static org.needle4j.configuration.PostConstructExecuteStrategy.ALWAYS;
+import static org.needle4j.configuration.PostConstructExecuteStrategy.DEFAULT;
+import static org.needle4j.configuration.PostConstructExecuteStrategy.NEVER;
+
+public class PostConstructExecuteStrategyTest {
+
+    @Test
+    public void fromString() {
```

Comment:
Would be nice to have a Parameterized Test here, now if "always" fails, we do not know if the rest would work, so you fix one and get the next error if you are lucky ...